known_servers/known_servers-production-v1.2.0.json

```diff
@@ -1,7 +1,7 @@
 {
     "active_servers": [
-        "transport.mainnet.raiden.anyblock.tools",
-        "transport.raiden.badgateway.tech"
+        "transport.raiden.badgateway.tech",
+        "transport.raiden.overdoze.se"
     ],
     "all_servers": [
         "transport.mainnet.raiden.anyblock.tools",
```

Comment:
Please also remove anyblock from the known servers list.

Reply:
Done.